aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -99,181 +107,195 @@ The preceding command adds the [aspnet-codegenerator scaffolding tool](xref:fund
+### Use SQLite for development, SQL Server for production
 
----
+When SQLite is selected, the template generated code is ready for development. The following code shows how to inject <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into `Startup`. `IWebHostEnvironment` is injected so `ConfigureServices` can use SQLite in development and SQL Server in production.
 
-The name of the connection string is passed in to the context by calling a method on a [DbContextOptions](/dotnet/api/microsoft.entityframeworkcore.dbcontextoptions) object. For local development, the [ASP.NET Core configuration system](xref:fundamentals/configuration/index) reads the connection string from the *appsettings.json* file.
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/StartupDevProd.cs?name=snippet_StartupClass&highlight=3,5,10,16-28)]
 
-<a name="cs"></a>
+# [Visual Studio for Mac](#tab/visual-studio-mac)
 
-## Add a database connection string
+Open a command window in the project directory. The project directory is the directory that contains the *Program.cs*, *Startup.cs*, and *.csproj* files.
 
-Add a connection string to the *appsettings.json* file:
+Export the scaffold tool path:
 
-# [Visual Studio](#tab/visual-studio)
+```console
+export PATH=$HOME/.dotnet/tools:$PATH
+```
 
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings.json?highlight=10-12)]
+Run the following command:
 
-# [Visual Studio Code / Visual Studio for Mac](#tab/visual-studio-code+visual-studio-mac)
+```dotnetcli
+dotnet-aspnet-codegenerator controller -name MoviesController -m Movie -dc MvcMovieContext --relativeFolderPath Controllers --useDefaultLayout --referenceScriptLibraries
+```
 
-[!code-json[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/appsettings_SQLite.json?highlight=10-12)]
+[!INCLUDE [explains scaffold generated params](~/includes/mvc-intro/model4.md)]
+
+<a name="sqlite-dev-vs-mac"></a>
+
+### Use SQLite for development, SQL Server for production
+
+When SQLite is selected, the template generated code is ready for development. The following code shows how to inject <xref:Microsoft.AspNetCore.Hosting.IWebHostEnvironment> into `Startup`. `IWebHostEnvironment` is injected so `ConfigureServices` can use SQLite in development and SQL Server in production.
+
+[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/StartupDevProd.cs?name=snippet_StartupClass&highlight=3,5,10,16-28)]
 
 ---
 
-Build the project as a check for compiler errors.
+<!-- End of tabs                  -->
 
-## Scaffold movie pages
+<a name="scaffolding-created"></a>
 
-Use the scaffolding tool to produce Create, Read, Update, and Delete (CRUD) pages for the movie model.
+Scaffolding creates:
 
-# [Visual Studio](#tab/visual-studio)
+* A movies controller: *Controllers/MoviesController.cs*.
+* Razor view files for Create, Delete, Details, Edit, and Index pages: `Views/Movies/\*.cshtml`.
```

Comment:
remove `\`

Reply:
![image](https://user-images.githubusercontent.com/3605364/116488760-dfb3da00-a82e-11eb-8c36-3d132b211adb.png)

over translating. Helps to look at other languages